### ISSUE TITLE
Entrust migration (ver. 5.2.x-dev): no default users table on migration file 

### ISSUE THREAD

**Issue by kungfu321** [Zizaco/entrust#652]
Hi, this is my first time using entrust.
when i'm doing php artisan entrust:migration and then php artisan migrate i got an error:
`←[37;41m
       ←[39;49m
←[37;41m  [Illuminate\Database\QueryException]
       ←[39;49m
←[37;41m  SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'role_u
ser'   ←[39;49m
←[37;41m  already exists (SQL: create table `role_user` (`user_id` int unsigned
not n  ←[39;49m
←[37;41m  ull, `role_id` int unsigned not null) default character set utf8 colla
te ut  ←[39;49m
←[37;41m  f8_unicode_ci)
       ←[39;49m
←[37;41m
       ←[39;49m

←[37;41m
       ←[39;49m
←[37;41m  [PDOException]
       ←[39;49m
←[37;41m  SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'role_u
ser'   ←[39;49m
←[37;41m  already exists
       ←[39;49m
←[37;41m
       ←[39;49m
`

`// Create table for storing roles
		Schema::create('roles', function (Blueprint $table) {
			$table->increments('id');
			$table->string('name')->unique();
			$table->string('display_name')->nullable();
			$table->string('description')->nullable();
			$table->timestamps();
		});`

**Comment by CDOLago** [comment]
The same thing happened to me. For some reason the `$usersTable` variable is empty when generating the migration. I was using a fresh Laravel 5.3 installation.

**Comment by aneeshsudhakaran** [comment]
Is this support Laravel 5.3?

**Comment by mcblum** [comment]
Same thing for me, @CDOLago.

**Comment by shikazu** [comment]
Any quick fix for this?

**Comment by mcblum** [comment]
@shikazu Yep! Either fill in the name of your users table manually in the migrations or make sure your config/auth.php has the following array, including the `table => users`.

```
   'providers' => [
        'users' => [
            'driver' => 'eloquent',
            'model' => App\User::class,
            'table' => 'users'
        ],
    ],
```
